nixos/modules/programs/projecteur.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.projecteur;
+in
+{
+  options.programs.projecteur = {
```

Comment:
I am unsure whether hardware or programs is more appropriate, for example the uhk configuration utility is also under [hardware.keyboard.uhk.enable](https://search.nixos.org/options?channel=23.05&from=0&size=50&sort=relevance&type=packages&query=uhk) .

Reply:
Good remark... I have no clue on that.

Reply:
I propose to merge it as it is, and do the change at a later point, if needed. WDYT ?

Reply:
fine for me but I cannot merge anyway and someone with merge rights might have an opinion